app/models/notification.rb

```diff
@@ -1,8 +1,17 @@
 class Notification < ApplicationRecord
   belongs_to :user
   belongs_to :organization
-  belongs_to :target, polymorphic: true
+  belongs_to :target, polymorphic: true, optional: true
 
+  enum subject: %i(
+    free_text
```

Comment:
Perhaps we could use the `custom`,   `announcement` or even `custom_announcement`, since this is is the main motivation of the free text feature.